homeassistant/components/alarm_control_panel/verisure.py

```diff
@@ -90,6 +90,10 @@ def alarm_arm_home(self, code=None):
         """Send arm home command."""
         set_arm_state('ARMED_HOME', code)
 
+    def alarm_arm_night(self, code=None):
+        """Send arm night command."""
+        set_arm_state('ARMED_HOME', code)
```

Comment:
This does not seem right. This is already the command for `alarm_arm_home`.

Reply:
Yeah, I wasn’t sure what to do because Verisure alarm system make no difference between night and home arming, in fact, there’s no such concept as night arming. So that makes it kind of the same thing in my opinion. Still we want to support the method because it is usable in HomeKit. So if we do not implement this method, we’ll get the exception in hass when executing night arming from the Home app with no status reply back to the Home app, making it loop the arming execution for some time. 

Do you have another idea how it should be done in order to somehow support the night arming in the Home app?

Reply:
Okay, we are not going to change a source implementation because some integration built on top needs a hack to function.

The correct way is to have HomeKit mark Verisure alarm control panels as not supporting arm night